Basic_data_types_and_operations_Part_2.py

```diff
@@ -0,0 +1,19 @@
+# Required task
+input_something = input("Input something - ")
+if input_something.isdigit():
+    if int(input_something) % 2 == 0:
+        print(f" The {input_something} is digit. And the number is even.")
+    else:
+        print(f" The {input_something} is digit. And the number is odd.")
+elif input_something.isalpha():
+    print(f" The {input_something} is word, its length is {len(input_something)}.")
+else:
+    print("Error")
+
+# Optional task
+input_something = input("Input something - ")
```

Comment:
the trick is that input data is always string, so it could be solved like 
case str() if user_input.isdigit()

Reply:
I thought about it.